proposals/ml_fit_class_support_vector.json

```diff
@@ -0,0 +1,153 @@
+{
+    "id": "ml_fit_class_support_vector",
+    "summary": "Train a support vector classification model",
+    "description": "Executes the fit of a support vector classification model based on training data. The support vector classification model aims to find a hyperplane that best separates different classes in the feature space.",
+    "categories": [
+        "machine learning"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "predictors",
+            "description": "The predictors for the classification model as a vector data cube. Aggregated to the features (vectors) of the target input variable.",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "datacube",
+                    "dimensions": [
+                        {
+                            "type": "geometry"
+                        },
+                        {
+                            "type": "bands"
+                        }
+                    ]
+                },
+                {
+                    "type": "object",
+                    "subtype": "datacube",
+                    "dimensions": [
+                        {
+                            "type": "geometry"
+                        },
+                        {
+                            "type": "other"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "target",
+            "description": "Labeled information for training the support vector classification model. Each entry corresponds to a set of features (predictors).",
```

Comment:
I still find this unclear. "Labeled information" can be interpreted as predictors plus a label, but I don't think that is intended here.
It's also risky  to use the term "features" (ML interpretation: predictor values) in another interpretation than used in the description of the predictors argument (EO interpretation: geometry). 

"Each entry corresponds to a set of .." is also a bit too loose as constraint, I think the point should be that the the `predictors` vector cube and `target` cube have a common geometry dimension, or at least that the backend is able to connect "predictor rows" from `predictors` with labels from `target`.